python/test/t_PhysicalSpaceCrossEntropyImportanceSampling_std.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python
+
+import openturns as ot
+import openturns.experimental as otexp
+from openturns.testing import assert_almost_equal
+ot.RandomGenerator.SetSeed(1)
+
+distribution_R = ot.LogNormalMuSigma(300.0, 30.0, 0.0).getDistribution()
+distribution_F = ot.Normal(75e3, 5e3)
+marginals = [distribution_R, distribution_F]
+distribution = ot.ComposedDistribution(marginals)
+
+model = ot.SymbolicFunction(['R', 'F'], ['R - F/(pi_*100.0)'])
+vect = ot.RandomVector(distribution)
+G = ot.CompositeRandomVector(model, vect)
```

Comment:
Variable names which have an upper case first letter are classes in Python, according to the PEP8.
```suggestion
g = ot.CompositeRandomVector(model, vect)
```